elife/nginx-php7.sls

```diff
@@ -32,6 +40,11 @@ php-fpm:
 
     service.running:
         - name: php7.0-fpm
+        - enable: True
         - require:
             - file: php-fpm
-            - file: php-fpm-config            
+            - file: php-fpm-config
+            - php-fpm-pool
+        - watch:
```

Comment:
also `pkg: php-nginx-deps` if the package version changes for some reason?

Reply:
Added.